src/modules/dashboard/components/SmiteDialog/SmiteDialog.tsx

```diff
@@ -126,6 +126,22 @@ const SmiteDialog = ({
     [totalReputationData],
   );
 
+  const { walletAddress: loggedInUserWalletAddress } = useLoggedInUser();
+
+  const selectedUser = useMemo(() => {
+    if (!colonyMembers) {
+      return undefined;
+    }
+
+    if (colonyMembers.subscribedUsers.length === 1) {
+      return colonyMembers.subscribedUsers[0];
+    }
+
+    return colonyMembers.subscribedUsers[0].id === loggedInUserWalletAddress
+      ? colonyMembers.subscribedUsers[1]
+      : colonyMembers.subscribedUsers[0];
```

Comment:
So this is just for clarity / better readability, not to do with your code logic (which is fine).

But I would have written this as:
```js

const [firstSubscriber, secondSubscriber] = colonyMembers?.subscribedUsers;

if (!secondSubscriber) {
  return firstSubscriber;
}

return firstSubscriber.profile.walletAddress === loggedInUserWalletAddress 
  ? secondSubscriber
  : firstSubscriber;
```

Reply:
done

Reply:
Actually, the first line should be:

```js
const [firstSubscriber, secondSubscriber] = colonyMembers?.subscribedUsers || [];
```

To have a fallback in case `colonyMembers?.subscribedUsers` returns `undefined`

Reply:
good catch! done